promtimer/cbstats.py

```diff
@@ -448,6 +449,74 @@ def get_user_log(cluster, user, password):
             logging.error('error: can\'t access cluster: {}'.format(err))
             return []
 
+class BackupStatsFiles(Source):
+    """
+    Represents a source of stats data that is a Prometheus instance running against
+    the stats data from cbbackupmgr runs.
+    """
+    def __init__(self, prometheus_tsdb_path, short_name, prometheus_port):
+        super(BackupStatsFiles, self).__init__(prometheus_port)
```

Comment:
IIRC, in python3 you can do:

 `super().__init__(prometheus_port)`

I _think_ that's true even for relatively old 3.x, not certain off hand.